### ISSUE TITLE
comma regression

### ISSUE THREAD

**Issue by jeremybmerrill** [tabulapdf/tabula-java#20]
In some circumstances, commas are moved to the end of a cell in "original"/"stream" mode.

```
$  -jar ./target/tabula-extractor-0.7.4-SNAPSHOT-jar-with-dependencies.jar ~/code/tabula-extractor/test/data/frx_2012_disclosure.pdf -a "158.355, 48.195, 182.835, 329.71500000000003" | tail -n1 | sed -r 's/\"([^\"]+)\".*/\1/'
AARON JOSHUA N , , 
```
it should be
```
AARON, JOSHUA, N
```

Presumably, something (either us or PDFBox) detects these as being on a different line and appends them to the end with spaces, as if they were on the line below.

**Comment by jeremybmerrill** [comment]
I should note all the fancy `sed` is just grabbing the first cell of the last line. :)

**Comment by jazzido** [comment]
Fixed by @melisabok in 721dac142b5767174f7e8208ab4a5fdfaa0f41c5
